src/Components/Home/Home.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { getCurrency } from '../../Redux/CurrencySlice';
+import './Home.css';
+import Currencies from '../Currencies/Currencies';
+import Search from '../Searcch/Search';
+import Button from '../Buttons/Button';
+import Loader from '../Loader/Loader';
+
+function Home() {
```

Comment:
- [ ] Kindly remove this back arrow from the Home page, it does nothing there and so should not be there. This is should only be in the details page.

  ![image](https://github.com/Peter-Dumbari/Metrics-Webapp/assets/81468675/e7f1b5ab-24b8-4bff-8718-5400c57fda07)

Reply:
@julie-ify, Thanks 👍  I will make required changes